router/xgress_edge/factory_test.go

```diff
@@ -107,7 +108,7 @@ func Test_load(t *testing.T) {
 const (
 	connectMaxQueuedConnections   = 50
 	connectMaxOutstandingConnects = 100
-	connectTimeoutMs              = 3000
+	connectTimeoutMs              = 1000 * time.Millisecond
 )
```

Comment:
I honestly cannot figure out why this was previously 3000 and passed. The code in the channel project had an int value of 1000. That value of 1000 was changed to a time.duration value do I've updated that here.